cypress/support/step_definitions/validations.md

```diff
@@ -412,3 +445,156 @@ While validating, if a key is present in both fcs-validation jsons (eg: cypress/
             }
         ]
 }
+
+
```

Comment:
Change Background into 

There are some test cases which involves calling firebolt methods with invalid parameters, missing parameters. Here, we are expecting error to be returned from the device. FCS has some default error types defined which can be used in various test cases. 

Default Validation Types in FCS
Below is the default error objects supported in FCS [errorContentObjects.json](https://github.com/rdkcentral/firebolt-certification-suite/blob/e97116c88ba79a2ad7f2fe23d55dc6101c3dd484/cypress/fixtures/objects/errorContentObjects.json)

NOT_SUPPORTED - This can be used for api's which are not supported
NOT_PERMITTED - This can be used for api's which are not supported
NOT_AVAILABLE - This can be used for api's which are not supported
INVALID_TYPE_PARAMS - This can be used for error validation which involves calling firebolt methods with invalid parameters, missing parameters etc.

Reply:
Remove the note

Reply:
addressed the comment